src/ui/prompts/commonPrompts.ts

```diff
@@ -20,7 +20,12 @@ export const keyPrompt: Prompt = {
     name: 'key',
     message: 'Key',
     suffix: ':',
-    transformer: hintTextTransformer('(Unique ID)'),
+    default: (answers: Record<string, string>) => {
+        if (answers.name) {
+            return answers.name.trim().replace(/\s+/g, '-').toLowerCase()
```

Comment:
during an update, don't we want to set the default as the current key since the name in kebab case might be different than the key?

Reply:
during an update it won't have the answer for the name as it won't have been prompted in succession of a name prompt

Reply:
the order of prompting has been swapped for features updates and variations update. It goes name and then key. But also, shouldn't we be returning the current key if it exists as the default instead of an empty string

Reply:
in `feature update` you choose to select the fields you're updating, name isn't guaranteed to be prompted if you're updating key. Although if you choose both name and key it will infer the key from the new name you set

Reply:
ahhh I see. I guess it makes sense to default the key to the new name if you're trying to update both key and name. 

Oh, I thought we were already defaulting to the previous value.